harness/determined/deploy/aws/cli.py

```diff
@@ -660,7 +659,6 @@ def handle_dump_master_config_template(args: argparse.Namespace) -> None:
                 Arg(
                     "--lore-version",
                     type=str,
-                    default="latest",
```

Comment:
Why does this got taken out?

Reply:
(we chatted offline)
we want the args to not be there so we can tell whether the user specified it explicitly or not.